app/src/main/java/com/jerboa/ui/components/person/PersonProfileLink.kt

```diff
@@ -34,14 +39,19 @@ fun PersonName(
     val style = MaterialTheme.typography.bodyMedium
 
     if (isPostCreator) {
-        Badge(
-            containerColor = MaterialTheme.colorScheme.tertiary,
+        Box(
```

Comment:
I will do just that, thanks!

Reply:
@dessalines Hey, I didnt change the other Badge() since it looks like they are being used as their intended use case, which is to show a number for like notifications. And not sure this badge should be affected by the corner radius changes since they are very small and I don't think there would be a way to make them slightly squared off, I feel the radius would make it look off-*circley*, kind of like a broken circle.

Reply:
Mmmk that's fine.